styles.css

```diff
@@ -0,0 +1,390 @@
+  --baloo-2-ff: "Baloo 2", sans-serif;
+
+  --font-regular: 400;
+  --font-medium: 500;
+  --font-semi-bold: 600;
+
+  --transition: 0.2s ease-in-out;
+}
+
+*,
+*::after,
+*::before {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: var(--baloo-2-ff);
+  font-size: 1rem;
+  font-weight: var(--font-regular);
+  line-height: 1.6;
+  color: var(--text-color);
+  min-width: 390px;
+}
+
+a {
+  display: inline-block;
+  text-decoration: none;
+  color: var(--text-color);
+}
+
+li {
+  list-style: none;
+}
+
+img {
+  display: block;
+  width: 100%;
+  height: auto;
+}
+
+h2 {
+  font-weight: var(--font-semi-bold);
+}
+
+/* Shared */
+
+.btn {
+  font-family: var(--baloo-2-ff);
+  font-size: 1rem;
+  font-weight: var(--font-semi-bold);
+  height: 50px;
+  width: 170px;
+  color: var(--main-background-color);
+  background: var(--accent-color);
+  border: none;
+  border-radius: 10px;
+  box-sizing: border-box;
+  transition: background var(--transition);
+}
+
+.btn:hover {
+  background: var(--hover-color);
+  cursor: pointer;
+}
+
+.btn--slim {
+  height: 40px;
+  width: 150px;
+}
+
+.btn--big {
+  width: 320px;
+}
+
+.list {
+  display: grid;
+  grid-auto-flow: column;
+  grid-auto-columns: max-content;
+}
+
+/* Layout */
+
+.container {
+  display: grid;
+  grid-template-columns: minmax(22px, 1fr) minmax(348px, 1718px) minmax(
+      22px,
+      1fr
+    );
+  grid-template-rows: 166px 1fr 137px;
+  grid-template-areas:
+    ". header ."
+    ". main ."
+    "footer footer footer";
+  justify-content: center;
+  min-height: 100vh;
+}
+
+/* Header */
+
+.header {
+  grid-area: header;
+  position: relative;
+  display: grid;
+  grid-template-columns: max-content max-content;
+  justify-content: space-between;
+  column-gap: 50px;
+  background-color: var(--main-background-color);
+  padding-top: 16px;
+}
+
+.header,
+.footer {
+  align-items: center;
+}
+
+.header__logo {
+  max-width: 165px;
+  cursor: pointer;
+}
+
+.header__bar {
+  display: grid;
+  grid-template-columns: max-content max-content;
+  place-items: center;
+  column-gap: 45px;
+}
+
+.header__navigation-list {
+  gap: 59px;
+}
+
+.header__navigation-link {
+  transition: scale var(--transition);
+}
+
+.header__navigation-link:hover {
+  transform: scale(1.1);
+}
+
+/* Burger */
+
+.header__burger {
+  position: relative;
+  top: -6px;
+  width: 36px;
+  height: 20px;
+  padding: 10px 0;
+  display: none;
+}
+
+.header__burger:hover {
+  cursor: pointer;
+}
+
+.header__burger-inner,
+.header__burger-inner::before,
+.header__burger-inner::after {
+  position: absolute;
+  display: block;
+  width: 36px;
+  height: 2.5px;
+  background: var(--text-color);
+  border-radius: 3px;
+  transition: transform var(--transition);
+}
+
+.header__burger-inner::before,
+.header__burger-inner::after {
+  content: "";
+}
+
+.header__burger-inner::before {
+  top: -8px;
+}
+
+.header__burger-inner::after {
+  bottom: -8px;
+}
+
+#menu-toggle {
```

Comment:
Avoid styling by ID whenever possible, as it carries higher specificity and can lead to issues in certain cases.
U can just add `class` and use it